.gitlab/ci/build.yml

```diff
@@ -574,6 +574,18 @@ build_example_motor_foc_knob:
   variables:
     EXAMPLE_DIR: examples/motor/foc_knob_example
 
+build_example_motor_servo:
```

Comment:
In addition to adding the servo component in `build.yml`, it is also necessary to add rules for servo in` rules.yml` to ensure that changes to servo and projects dependent on servo can trigger the CI. You can refer to any existing component in `rules.yml` as a reference to complete the rule definition for servo.

Reply:
I did but I'm not sure if I did correctly. Can you kindly review it?
Thank you, @YanKE01.